locales/en.json

```diff
@@ -1358,6 +1358,7 @@
         "message": "People will be able to buy your NFT."
       }
     },
+    "notRecommendedModify": "Please note that this value is not recommended to be modified as it may lead to misuse and confusion",
```

Comment:
```suggestion
    "notRecommendedModify": "Please note that this field is not recommended to be modified as it may lead to misuse and confusion",
```

Reply:
make sense. updated